src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,34 @@
+package core.basesyntax;
+
+public class RightTriangle extends Default {
+    private double size;
+    private double firstLeg;
+    private double secondLeg;
+
+    public RightTriangle(String color, double size, double firstLeg, double secondLeg) {
+        super(color);
+        this.size = size;
+        this.firstLeg = firstLeg;
+        this.secondLeg = secondLeg;
+    }
+
+    @Override
+    public double getArea() {
+
+        return size * size;
```

Comment:
The calculation of the area for a right triangle should be `0.5 * firstLeg * secondLeg` instead of `size * size`. This seems to be an error in the logic.